### PR TITLE
Bootstrap CSS to Tailwind CSS

### PR DESCRIPTION
**Pull request by kaseyuu**
Migrate from Bootstrap CSS to Tailwind CSS for Login page.

### CODE REVIEW

File: front_end_project/src/pages/login/login.js
Comment (mukundnivas) [inline front_end_project/src/pages/login/login.js:1]: Hey Kasey,

This seems to be the same issue from PR #105. Please refer to it.

The Login page still seems to contain all the React-bootstrap higher order classes. I could you have refactored couple of class names to Tailwind supported ones. The requirement here to scratch all the React-bootstrap's higher-order classes like "Button" etc to Tailwind CSS's utility classes.

**Review by mukundnivas — CHANGES_REQUESTED** [review]
This seems to be the same issue from PR #105. Please refer to it.

File: front_end_project/src/pages/login/login.js
Comment (kaseyuu) [reply front_end_project/src/pages/login/login.js:1]: Hi Mukund, just to confirm, do you expect us to completely remove React Bootstrap and replace it with HTML+Tailwind CSS?

Comment (mukundnivas) [reply front_end_project/src/pages/login/login.js:1]: Yes @kaseyuu porting over to Tailwind means porting all the components to utility based classes from Tailwind CSS.

**Comment by mukundnivas** [comment]
As discussed, please port all the components of the page to Tailwind CSS.

**Pull request closed without merging**